tests/test_date_conversions.py

```diff
@@ -55,8 +56,50 @@ def test_return_periods(requests_mock, flask_test_client):
     assert output_3["from-year"] == ["2022/2023", "2023/2024"]
 
 
-def test_quarter_to_dates():
-    assert quarter_to_date(quarter="2", year="2022/2023") == "2022-07-01T00:00:00Z"
-    assert quarter_to_date(quarter="1", year="2020/2021") == "2020-04-01T00:00:00Z"
-    assert quarter_to_date(quarter="3", year="2019/2020") == "2019-10-01T00:00:00Z"
-    assert quarter_to_date(quarter="4", year="2021/2022") == "2021-01-01T00:00:00Z"
+def test_financial_quarter_from_mapping():
+    # Q1 includes April, May, June
+    assert (
+        financial_quarter_from_mapping(quarter="1", year="2020/2021")
+        == "2020-04-01T00:00:00Z"
+    )
+
+    # Q2 includes July, August, September
+    assert (
+        financial_quarter_from_mapping(quarter="2", year="2022/2023")
+        == "2022-07-01T00:00:00Z"
+    )
+
+    # Q3 includes October, November, December
+    assert (
+        financial_quarter_from_mapping(quarter="3", year="2019/2020")
+        == "2019-10-01T00:00:00Z"
+    )
+
+    # January, February, March - selecting Q4 should return up to January of the next year
```

Comment:
these comments don't make sense anymore, in the current context, as this test doesn't return a range, just a single date